src/entities/transformers/default-boolean.transformer.ts

```diff
@@ -0,0 +1,13 @@
+import { ValueTransformer } from 'typeorm';
+
+export class DefaultBooleanTransformer implements ValueTransformer {
+  constructor(private readonly defaultValue: boolean) {}
+
+  public from(value?: number): boolean {
+    return !!value;
+  }
+
+  public to(value?: boolean | undefined): number {
```

Comment:
```suggestion
  public to(value?: boolean): number {
```
? 가 이미 undefined일 수 있다는 의미이기 때문에 중복으로 타이핑된듯합니다.